File: Ritika kumar/Basicsofprogramming/Gradingsystem.java
import java.util.Scanner;

public class Gradingsystem {
    
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        int marks = s.nextInt();


        if(marks>90)
        {
            System.out.println("excellent");
        }

        else if(marks>80 && marks<=90)
        {
            System.out.println("good");
        }

        else if(marks>70 && marks<=80)
        {
            System.out.println("fair");
        }

        else if(marks>60 && marks<=70)
        {
            System.out.println("meets expectations");
        }

        else
       {
           System.out.println("below par");
       }
    }
}

File: Oishi/Basics of Programming/Getting Started/GradingSystem.cpp
#include<iostream>
using namespace std;
int main(int argc, char**argv){
    int marks;
    cin>>marks;
    
    //write your code here
    if (marks > 90) cout << "excellent" << endl;
    else if (marks <=90 && marks > 80) cout << "good" << endl;
    else if (marks <= 80 && marks > 70) cout << "fair" << endl;
    else if (marks <=70 && marks > 60) cout << "meets expectations" << endl;
    else cout << "below par" << endl;
}